### DISCUSSION TITLE
Why is useFetch covered?

### DISCUSSION THREAD

**Discussion by FatNerdPeng** [nuxt/nuxt#18726] [Questions]
Hello,I have a strange problem following:
![image](https://user-images.githubusercontent.com/33616855/216505322-570ce42f-85db-4bc3-b198-e0790c667202.png)
![image](https://user-images.githubusercontent.com/33616855/216505472-61ba749f-7b83-4fdb-9e8c-6930dd6ce584.png)
![image](https://user-images.githubusercontent.com/33616855/216505632-7f0ce52a-4ac5-4b55-a83e-9b8082833641.png)
As shown above,I need to make two requests in the function.In order to show my problem conveniently, I have simplified the request here, which does not affect the questions I ask.
When I refresh the page,the result is following:
![image](https://user-images.githubusercontent.com/33616855/216506503-6f9ddd07-88b1-43f2-b5dd-c34fc6efec3d.png)
So,where is the request 'aaa'?Now I'll change their positions:
![image](https://user-images.githubusercontent.com/33616855/216506882-283778ec-526e-4679-8377-9c0f336fc2d9.png)
![image](https://user-images.githubusercontent.com/33616855/216506920-0776411c-81e8-49a2-a802-5c6bf38533ca.png)
So why?In my actual development, this problem caused me to be unable to obtain the data I need through the first request,I want to know what I did wrong?

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

~Given the reproduction, the problem is that you do not `await` the async `useGetPageData` composable.~
Cause: https://github.com/nuxt/nuxt/issues/14723

**Reply by FatNerdPeng** [reply]
No problem, I have finished this task https://github.com/FatNerdPeng/github-nolx34.git

**Reply by TheAlexLichter** [reply]
You do not `await` the async `useGetPageData` composable.

**Reply by FatNerdPeng** [reply]
Thanks,I tried as you said, but I still can't.I found that an article was the same as the problem I encountered https://github.com/nuxt/nuxt/discussions/15941
I think the thinking may be in here?

**Reply by TheAlexLichter** [reply]
After going further down, this is the same as in https://github.com/nuxt/nuxt/issues/14725 -> See https://github.com/nuxt/nuxt/issues/14723 and https://github.com/nuxt/nuxt/issues/12774#issuecomment-1397241245

**Reply by FatNerdPeng** [reply]
Ok, thank you. I will continue to follow up on https://github.com/nuxt/nuxt/issues/14723 and https://github.com/nuxt/nuxt/issues/12774#issuecomment-1397241245
